### ISSUE TITLE
bundler: failed to load command: fastlane (/usr/local/bin/fastlane)

### ISSUE THREAD

**Issue by ip19** [lyndsey-ferguson/fastlane-plugin-test_center#136]
#136   An issue on the stage of rerunning tests again

#### Question Description
1. I'm executing ' bundle exec fastlane test'
2. Everything goes ok but I get an error when rerunning tests
![Screen Shot 2019-08-30 at 16 32 57](https://user-images.githubusercontent.com/37904377/64025050-7d2fa580-cb44-11e9-8837-5fe2f41e59b8.png)

![Screen Shot 2019-08-30 at 16 40 09](https://user-images.githubusercontent.com/37904377/64025207-d8619800-cb44-11e9-8c74-84aead2bed71.png)

**Comment by lyndsey-ferguson** [comment]
@ip19 can you please provide this information as part of the Issue?

```
##### Complete output when running fastlane, including the stack trace and command used
<!-- You can use: `--verbose --capture_output` as the last commandline arguments to get that collected for you -->

<!-- The output of `--verbose --capture_output` could contain sensitive data such as application ids, certificate ids, passwords, authentication tokens, or email addreses, Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue -->

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<!-- Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used the `--capture_output` option, please remove this block as it is already included there. -->

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>
##### Complete output when running fastlane, including the stack trace and command used
<!-- You can use: `--verbose --capture_output` as the last commandline arguments to get that collected for you -->

<!-- The output of `--verbose --capture_output` could contain sensitive data such as application ids, certificate ids, passwords, authentication tokens, or email addreses, Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue -->

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<!-- Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used the `--capture_output` option, please remove this block as it is already included there. -->

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>
```

**Comment by ip19** [comment]
<details>
  <pre>[epuavinw009$ fastlane run --verbose --capture_output
[✔] 🚀 
[16:22:44]: fastlane detected a Gemfile in the current directory
[16:22:44]: however it seems like you don't use `bundle exec`
[16:22:44]: to launch fastlane faster, please use
[16:22:44]: 
[16:22:44]: $ bundle exec fastlane run --verbose --capture_output
[16:22:44]: 
[16:22:44]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
DEBUG [2019-08-31 16:22:45.35]: Checking if there are any plugins that should be loaded...
DEBUG [2019-08-31 16:22:45.40]: Loading 'fastlane-plugin-test_center' plugin
INFO [2019-08-31 16:22:45.43]: gem 'fastlane-plugin-test_center' is already installed
DEBUG [2019-08-31 16:22:47.60]: All plugins are up to date
epuavinw009$ ]</pre>
</details>

**Comment by ip19** [comment]
> @ip19 can you please provide this information as part of the Issue?
> 
> ```
> ##### Complete output when running fastlane, including the stack trace and command used
> 
> 
> 
> 
> <details>
>   <pre>[INSERT OUTPUT HERE]</pre>
> </details>
> 
> ### Environment
> 
> 
> 
> <details>
>   <pre>[INSERT OUTPUT HERE]</pre>
> </details>
> ##### Complete output when running fastlane, including the stack trace and command used
> 
> 
> 
> 
> <details>
>   <pre>[INSERT OUTPUT HERE]</pre>
> </details>
> 
> ### Environment
> 
> 
> 
> <details>
>   <pre>[INSERT OUTPUT HERE]</pre>
> </details>
> ```

**Comment by lyndsey-ferguson** [comment]
@ip19 Can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again with `--verbose`?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-136-report-collator-crash"
```

If it still crashes, please attach the console output as a text file to this issue (makes it easier for me to review). If it does not crash, please let me know.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.5](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/tree/v3.8.5)
